 2019-05-13 20:28:09,800 [P29660/D2/T1] INFO  Umbraco.Core.CoreBootManager - Umbraco 7.14.0 application starting on DESKTOP-2F2DTR9
 2019-05-13 20:28:09,829 [P29660/D2/T1] INFO  Umbraco.Core.PluginManager - Determining hash of code files on disk
 2019-05-13 20:28:09,930 [P29660/D2/T1] INFO  Umbraco.Core.PluginManager - Hash determined (took 100ms)
 2019-05-13 20:28:09,988 [P29660/D2/T1] INFO  Umbraco.Core.MainDom - Acquiring MainDom...
 2019-05-13 20:28:09,989 [P29660/D2/T1] INFO  Umbraco.Core.MainDom - Acquired MainDom.
 2019-05-13 20:28:09,990 [P29660/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-05-13 20:28:10,353 [P29660/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 362ms)
 2019-05-13 20:28:10,354 [P29660/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IApplicationStartupHandler
 2019-05-13 20:28:10,359 [P29660/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IApplicationStartupHandler (took 5ms)
 2019-05-13 20:28:10,376 [P29660/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-05-13 20:28:10,376 [P29660/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-05-13 20:28:10,376 [P29660/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IPropertyEditorValueConverter
 2019-05-13 20:28:10,382 [P29660/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IPropertyEditorValueConverter (took 5ms)
 2019-05-13 20:28:10,382 [P29660/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-05-13 20:28:10,383 [P29660/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-05-13 20:28:10,383 [P29660/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IPropertyValueConverter
 2019-05-13 20:28:10,388 [P29660/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IPropertyValueConverter (took 5ms)
 2019-05-13 20:28:10,399 [P29660/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-05-13 20:28:10,400 [P29660/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-05-13 20:28:10,400 [P29660/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Web.Mvc.SurfaceController
 2019-05-13 20:28:10,406 [P29660/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Web.Mvc.SurfaceController (took 7ms)
 2019-05-13 20:28:10,407 [P29660/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-05-13 20:28:10,407 [P29660/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-05-13 20:28:10,407 [P29660/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Web.WebApi.UmbracoApiController
 2019-05-13 20:28:10,413 [P29660/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Web.WebApi.UmbracoApiController (took 5ms)
 2019-05-13 20:28:11,880 [P29660/D2/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = True
 2019-05-13 20:28:12,317 [P29660/D2/T1] INFO  Umbraco.Web.Cache.CacheRefresherEventHandler - Initializing Umbraco internal event handlers for cache refreshing
 2019-05-13 20:28:12,466 [P29660/D2/T1] INFO  Umbraco.Web.Search.ExamineEvents - Initializing Examine and binding to business logic events
 2019-05-13 20:28:12,466 [P29660/D2/T1] INFO  Umbraco.Web.Search.ExamineEvents - Adding examine event handlers for index providers: 3
 2019-05-13 20:28:12,598 [P29660/D2/T1] WARN  Umbraco.Core.Sync.DatabaseServerMessenger - No last synced Id found, this generally means this is a new server/install. The server will build its caches and indexes, and then adjust its last synced Id to the latest found in the database and maintain cache updates based on that Id.
 2019-05-13 20:28:12,703 [P29660/D2/T1] INFO  umbraco.content - Load Xml from file...
 2019-05-13 20:28:12,763 [P29660/D2/T1] INFO  umbraco.content - Loaded Xml from file.
 2019-05-13 20:28:12,767 [P29660/D2/T1] INFO  umbraco.content - Loading content from database...
 2019-05-13 20:28:14,168 [P29660/D2/T1] INFO  UmbracoExamine.DataServices.UmbracoLogService - Rebuilding index, Provider=InternalMemberIndexer, NodeId=-1
 2019-05-13 20:28:14,228 [P29660/D2/T1] INFO  UmbracoExamine.DataServices.UmbracoLogService - PerformIndexAll - Start data queries - member, Provider=InternalMemberIndexer, NodeId=-1
 2019-05-13 20:28:14,873 [P29660/D2/T1] INFO  UmbracoExamine.DataServices.UmbracoLogService - PerformIndexAll - End data queries - member, took 645ms, Provider=InternalMemberIndexer, NodeId=-1
 2019-05-13 20:28:14,874 [P29660/D2/T1] INFO  UmbracoExamine.DataServices.UmbracoLogService - Rebuilding index, Provider=InternalIndexer, NodeId=-1
 2019-05-13 20:28:14,892 [P29660/D2/T1] INFO  UmbracoExamine.DataServices.UmbracoLogService - PerformIndexAll - Start data queries - content, Provider=InternalIndexer, NodeId=-1
 2019-05-13 20:28:15,867 [P29660/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-05-13 20:28:15,867 [P29660/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-05-13 20:28:15,867 [P29660/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IParameterEditor
 2019-05-13 20:28:15,873 [P29660/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IParameterEditor (took 5ms)
 2019-05-13 20:28:16,080 [P29660/D2/T1] INFO  UmbracoExamine.DataServices.UmbracoLogService - PerformIndexAll - End data queries - content, took 1187ms, Provider=InternalIndexer, NodeId=-1
 2019-05-13 20:28:16,080 [P29660/D2/T1] INFO  UmbracoExamine.DataServices.UmbracoLogService - PerformIndexAll - Start data queries - media, Provider=InternalIndexer, NodeId=-1
 2019-05-13 20:28:16,579 [P29660/D2/T1] INFO  UmbracoExamine.DataServices.UmbracoLogService - PerformIndexAll - End data queries - media, took 498ms, Provider=InternalIndexer, NodeId=-1
 2019-05-13 20:28:16,579 [P29660/D2/T1] INFO  UmbracoExamine.DataServices.UmbracoLogService - Rebuilding index, Provider=ExternalIndexer, NodeId=-1
 2019-05-13 20:28:16,602 [P29660/D2/T1] INFO  UmbracoExamine.DataServices.UmbracoLogService - PerformIndexAll - Start data queries - content, Provider=ExternalIndexer, NodeId=-1
 2019-05-13 20:28:17,173 [P29660/D2/T1] INFO  UmbracoExamine.DataServices.UmbracoLogService - PerformIndexAll - End data queries - content, took 571ms, Provider=ExternalIndexer, NodeId=-1
 2019-05-13 20:28:17,174 [P29660/D2/T1] INFO  UmbracoExamine.DataServices.UmbracoLogService - PerformIndexAll - Start data queries - media, Provider=ExternalIndexer, NodeId=-1
 2019-05-13 20:28:17,361 [P29660/D2/T1] INFO  UmbracoExamine.DataServices.UmbracoLogService - PerformIndexAll - End data queries - media, took 186ms, Provider=ExternalIndexer, NodeId=-1
 2019-05-13 20:28:17,494 [P29660/D2/T1] INFO  Hangfire.SqlServer.SqlServerObjectsInstaller - Start installing Hangfire SQL objects...
 2019-05-13 20:28:17,724 [P29660/D2/T1] INFO  Hangfire.SqlServer.SqlServerObjectsInstaller - Hangfire SQL objects installed.
 2019-05-13 20:28:17,743 [P29660/D2/T1] INFO  Hangfire.BackgroundJobServer - Starting Hangfire Server using job storage: 'SQL Server: 188.121.44.214@Nomad'
 2019-05-13 20:28:17,744 [P29660/D2/T1] INFO  Hangfire.BackgroundJobServer - Using the following options for SQL Server job storage: Queue poll interval: 00:00:15.
 2019-05-13 20:28:17,744 [P29660/D2/T1] INFO  Hangfire.BackgroundJobServer - Using the following options for Hangfire Server:
    Worker count: 20
    Listening queues: 'default'
    Shutdown timeout: 00:00:15
    Schedule polling interval: 00:00:15
 2019-05-13 20:28:17,840 [P29660/D2/TBackgroundServerProcess #1] INFO  Hangfire.Server.BackgroundServerProcess - Server desktop-2f2dtr9:29660:ab735ec4 successfully announced in 70,4995 ms
 2019-05-13 20:28:17,847 [P29660/D2/TBackgroundServerProcess #1] INFO  Hangfire.Server.BackgroundServerProcess - Server desktop-2f2dtr9:29660:ab735ec4 is starting the registered dispatchers: ServerWatchdog, ServerJobCancellationWatcher, ExpirationManager, CountersAggregator, Worker, DelayedJobScheduler, RecurringJobScheduler...
 2019-05-13 20:28:18,046 [P29660/D2/TBackgroundServerProcess #1] INFO  Hangfire.Server.BackgroundServerProcess - Server desktop-2f2dtr9:29660:ab735ec4 all the dispatchers started
 2019-05-13 20:28:18,097 [P29660/D2/T1] INFO  Umbraco.Core.CoreBootManager - Umbraco application startup complete (took 8448ms)
 2019-05-13 20:28:18,186 [P29660/D2/T10] INFO  umbraco.content - Save Xml to file...
 2019-05-13 20:28:18,212 [P29660/D2/T10] INFO  umbraco.content - Saved Xml to file.
 2019-05-13 20:28:18,365 [P29660/D2/T10] INFO  Umbraco.Core.Sync.ApplicationUrlHelper - New ApplicationUrl detected: http://localhost:58484/umbraco
 2019-05-13 20:28:18,366 [P29660/D2/T10] INFO  Umbraco.Core.Sync.ApplicationUrlHelper - ApplicationUrl: http://localhost:58484/umbraco (UmbracoModule request)
 2019-05-13 20:28:23,183 [P29660/D2/T10] WARN  Umbraco.Web.Mvc.RenderRouteHandler - The current Document Type home matches a locally declared controller of type Nomad.Controllers.HomeController. Custom Controllers for Umbraco routing must implement 'Umbraco.Web.Mvc.IRenderController' and inherit from 'System.Web.Mvc.ControllerBase'.
 2019-05-13 20:28:26,890 [P29660/D2/T10] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-05-13 20:28:26,890 [P29660/D2/T10] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-05-13 20:28:26,891 [P29660/D2/T10] INFO  Umbraco.Core.PluginManager - Resolving [Umbraco.Core.Persistence.Mappers.MapperForAttribute]Umbraco.Core.Persistence.Mappers.BaseMapper
 2019-05-13 20:28:26,897 [P29660/D2/T10] INFO  Umbraco.Core.PluginManager - Resolved [Umbraco.Core.Persistence.Mappers.MapperForAttribute]Umbraco.Core.Persistence.Mappers.BaseMapper (took 6ms)
 2019-05-13 20:28:27,371 [P29660/D2/T10] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-05-13 20:28:27,371 [P29660/D2/T10] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-05-13 20:28:27,371 [P29660/D2/T10] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.Deploy.IServiceConnector
 2019-05-13 20:28:27,378 [P29660/D2/T10] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.Deploy.IServiceConnector (took 6ms)
 2019-05-13 20:32:09,118 [P29660/D2/T46] ERROR Umbraco.Web.Strategies.ServerRegistrationEventHandler - Failed to update server record in database.
System.InvalidOperationException: Upłynął limit czasu. Limit upłynął przed uzyskaniem połączenia z puli. Mogło to nastąpić z powodu użycia całej puli połączeń i osiągnięcia maksymalnego rozmiaru puli.
   w System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   w System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   w System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   w System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   w System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   w System.Data.SqlClient.SqlConnection.Open()
   w Umbraco.Core.Persistence.FaultHandling.RetryPolicy.<>c__DisplayClass20_0.<ExecuteAction>b__0()
   w Umbraco.Core.Persistence.FaultHandling.RetryPolicy.ExecuteAction[TResult](Func`1 func)
   w Umbraco.Core.Persistence.FaultHandling.RetryPolicy.ExecuteAction(Action action)
   w Umbraco.Core.Persistence.PetaPocoConnectionExtensions.OpenWithRetry(IDbConnection connection, RetryPolicy retryPolicy)
   w Umbraco.Core.Persistence.PetaPocoConnectionExtensions.OpenWithRetry(IDbConnection connection)
   w Umbraco.Core.Persistence.Database.OpenSharedConnection()
   w Umbraco.Core.Persistence.Database.BeginTransaction(IsolationLevel isolationLevel)
   w Umbraco.Core.Scoping.Scope.get_Database()
   w Umbraco.Core.Persistence.UnitOfWork.ScopeUnitOfWork.get_Database()
   w Umbraco.Core.Persistence.LockingRepository`1.WithWriteLocked(Action`1 action, Boolean autoCommit)
   w Umbraco.Core.Services.ServerRegistrationService.TouchServer(String serverAddress, String serverIdentity, TimeSpan staleTimeout)
   w Umbraco.Web.Strategies.ServerRegistrationEventHandler.TouchServerTask.PerformRun()
 2019-05-13 20:32:09,297 [P29660/D2/T63] ERROR Umbraco.Web.BatchedDatabaseServerMessenger+InstructionProcessing - Failed (will repeat).
System.Data.SqlClient.SqlException (0x80131904): Klient nie może ustanowić połączenia z powodu błędu podczas procesu inicjowania połączenia, który wystąpił przed logowaniem. Prawdopodobne przyczyny są następujące: klient próbował nawiązać połączenie z nieobsługiwaną wersją programu SQL Server; serwer był zbyt zajęty, aby akceptować nowe połączenia; wystąpiło ograniczenie zasobów (za mało pamięci lub maksymalna dozwolona liczba połączeń) na serwerze. (provider: TCP Provider, error: 0 - Istniejące połączenie zostało gwałtownie zamknięte przez zdalnego hosta.) ---> System.ComponentModel.Win32Exception (0x80004005): Istniejące połączenie zostało gwałtownie zamknięte przez zdalnego hosta
   w System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   w System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   w System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   w System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   w System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   w System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   w System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   w System.Data.SqlClient.SqlConnection.Open()
   w Umbraco.Core.Persistence.FaultHandling.RetryPolicy.<>c__DisplayClass20_0.<ExecuteAction>b__0()
   w Umbraco.Core.Persistence.FaultHandling.RetryPolicy.ExecuteAction[TResult](Func`1 func)
   w Umbraco.Core.Persistence.FaultHandling.RetryPolicy.ExecuteAction(Action action)
   w Umbraco.Core.Persistence.PetaPocoConnectionExtensions.OpenWithRetry(IDbConnection connection, RetryPolicy retryPolicy)
   w Umbraco.Core.Persistence.PetaPocoConnectionExtensions.OpenWithRetry(IDbConnection connection)
   w Umbraco.Core.Persistence.Database.OpenSharedConnection()
   w Umbraco.Core.Persistence.Database.BeginTransaction(IsolationLevel isolationLevel)
   w Umbraco.Core.Scoping.Scope.get_Database()
   w Umbraco.Core.DatabaseContext.get_Database()
   w Umbraco.Core.Sync.DatabaseServerMessenger.ProcessDatabaseInstructions()
   w Umbraco.Core.Sync.DatabaseServerMessenger.Sync()
   w Umbraco.Web.BatchedDatabaseServerMessenger.InstructionProcessing.TryPerformRun()
   w Umbraco.Web.BatchedDatabaseServerMessenger.InstructionProcessing.PerformRun()
ClientConnectionId:884a4e74-dc45-4e46-96fd-ba7370190516
Error Number:10054,State:0,Class:20
 2019-05-13 20:32:25,461 [P29660/D2/TRecurringJobScheduler #1] INFO  Hangfire.Processing.BackgroundExecution - Execution RecurringJobScheduler recovered from the Faulted state after 00:00:16.1641107 and is in the Running state now
 2019-05-13 20:32:25,484 [P29660/D2/TDelayedJobScheduler #1] INFO  Hangfire.Processing.BackgroundExecution - Execution DelayedJobScheduler recovered from the Faulted state after 00:00:16.1868712 and is in the Running state now
 2019-05-13 20:33:15,330 [P29660/D2/T10] WARN  Umbraco.Web.Mvc.RenderRouteHandler - The current Document Type home matches a locally declared controller of type Nomad.Controllers.HomeController. Custom Controllers for Umbraco routing must implement 'Umbraco.Web.Mvc.IRenderController' and inherit from 'System.Web.Mvc.ControllerBase'.
 2019-05-13 20:34:21,303 [P29660/D2/TWorker #20] ERROR Hangfire.Processing.BackgroundExecution - Execution Worker is in the Failed state now due to an exception, execution will be retried no more than in 00:00:25
System.Data.SqlClient.SqlException (0x80131904): Upłynął limit czasu połączenia. Limit czasu upłynął podczas próby wykorzystania potwierdzenia uzgadniania wykonanego przed logowaniem. Mogło to być spowodowane niepowodzeniem uzgadniania przed logowaniem lub tym, że serwer nie mógł na czas odpowiedzieć.  Zużycie czasu podczas próby nawiązania połączenia z tym serwerem: [przed logowaniem] inicjowanie=56311; uzgadnianie=0;  ---> System.ComponentModel.Win32Exception (0x80004005): Upłynął limit czasu operacji oczekiwania
   w System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   w System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   w System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   w System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   w System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   w System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   w System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   w System.Data.SqlClient.SqlConnection.Open()
   w Hangfire.SqlServer.SqlServerStorage.CreateAndOpenConnection()
   w Hangfire.SqlServer.SqlServerJobQueue.DequeueUsingTransaction(String[] queues, CancellationToken cancellationToken)
   w Hangfire.SqlServer.SqlServerJobQueue.Dequeue(String[] queues, CancellationToken cancellationToken)
   w Hangfire.SqlServer.SqlServerConnection.FetchNextJob(String[] queues, CancellationToken cancellationToken)
   w Hangfire.Server.Worker.Execute(BackgroundProcessContext context)
   w Hangfire.Server.BackgroundProcessDispatcherBuilder.ExecuteProcess(Guid executionId, Object state)
   w Hangfire.Processing.BackgroundExecution.Run(Action`2 callback, Object state)
ClientConnectionId:6730843e-a4d9-4bfa-9b8f-6104b24f6142
Error Number:-2,State:0,Class:11
 2019-05-13 20:34:37,374 [P29660/D2/TRecurringJobScheduler #1] INFO  Hangfire.Processing.BackgroundExecution - Execution RecurringJobScheduler recovered from the Faulted state after 00:00:16.1017305 and is in the Running state now
 2019-05-13 20:34:37,396 [P29660/D2/TDelayedJobScheduler #1] INFO  Hangfire.Processing.BackgroundExecution - Execution DelayedJobScheduler recovered from the Faulted state after 00:00:16.1240841 and is in the Running state now
 2019-05-13 20:35:02,496 [P29660/D2/T46] WARN  Umbraco.Web.Mvc.RenderRouteHandler - The current Document Type home matches a locally declared controller of type Nomad.Controllers.HomeController. Custom Controllers for Umbraco routing must implement 'Umbraco.Web.Mvc.IRenderController' and inherit from 'System.Web.Mvc.ControllerBase'.
 2019-05-13 20:36:03,327 [P29660/D2/TWorker #1] INFO  Hangfire.Processing.BackgroundExecution - Execution Worker recovered from the Failed state after 00:01:42.0179675 and is in the Running state now
 2019-05-13 20:42:05,106 [P29660/D2/T51] WARN  Umbraco.Web.Mvc.RenderRouteHandler - The current Document Type home matches a locally declared controller of type Nomad.Controllers.HomeController. Custom Controllers for Umbraco routing must implement 'Umbraco.Web.Mvc.IRenderController' and inherit from 'System.Web.Mvc.ControllerBase'.
 2019-05-13 20:44:39,419 [P29660/D2/T59] WARN  Umbraco.Web.Mvc.RenderRouteHandler - The current Document Type home matches a locally declared controller of type Nomad.Controllers.HomeController. Custom Controllers for Umbraco routing must implement 'Umbraco.Web.Mvc.IRenderController' and inherit from 'System.Web.Mvc.ControllerBase'.
 2019-05-13 20:46:38,119 [P29660/D2/T66] WARN  Umbraco.Web.Mvc.RenderRouteHandler - The current Document Type home matches a locally declared controller of type Nomad.Controllers.HomeController. Custom Controllers for Umbraco routing must implement 'Umbraco.Web.Mvc.IRenderController' and inherit from 'System.Web.Mvc.ControllerBase'.
